src/dscanner/analysis/style.d

```diff
@@ -232,5 +235,10 @@ unittest
 			     ^^^^^^^^^^^^^^ [warn]: Function name 'WinButWithBody' does not match style guidelines. +/
 	}c, sac);
 
+	assertAnalyzerWarnings(q{
+		@("nolint(style_check)")
+		module AMODULE;
```

Comment:
```suggestion
		module mymod;
		int BAD_NAME;
```

ah the idea was here that the nolint would apply to the entire file, which is also what it currently does with the implementation. This test kind of implies that it would only apply to the module declaration itself and not the other code below though.

I think it's desirable to have the things apply to the whole file, so can you change the test to better reflect that? e.g. create a variable that has broken style name, but keep the module name valid.

For single instance checks like this we will probably want to apply the `nolint(...)` parsing to comments on the same line of the error as well, but that'd be for a different PR.